todo-app/app/views/tasks/index.html.haml

```diff
@@ -0,0 +1,20 @@
+%h1 Todo List
+
+%table{ class: 'table' }
+  %tr
+    %th Title
+    %th Details
+    %th Complete?
+
+  - @tasks.each do |t|
+    %tr
+      %td= t.title
+      %td
+        %a{ href: "/tasks/#{t.id}" } Details
```

Comment:
I'm pretty sure `task_path` is right. There's an extra level of Railsiness available though, using the `task` instance as the destination: `%td= link_to 'Details', t`